src/app/main/main.html

```diff
@@ -66,4 +72,42 @@
     </bookmark-list>
   </modal>
   <dataset-modal></dataset-modal>
+  <tour step="currentStep">
+    <virtual-step
```

Comment:
This may be overkill, but it might make maintenance easier to require this markup in via an [ng-include](https://docs.angularjs.org/api/ng/directive/ngInclude)